clang/test/Driver/sycl-deprecated.cpp

```diff
@@ -1,8 +1,13 @@
 /// Test for any deprecated options
 // RUN: %clangxx -fsycl-explicit-simd %s -### 2>&1 | FileCheck %s -DOPTION=-fsycl-explicit-simd
 // RUN: %clangxx -fno-sycl-explicit-simd %s -### 2>&1 | FileCheck %s -DOPTION=-fno-sycl-explicit-simd
-// RUN: %clangxx -fsycl -fsycl-use-bitcode %s -### 2>&1 | FileCheck %s -DOPTION=-fsycl-use-bitcode
-// RUN: %clangxx -fsycl -fno-sycl-use-bitcode %s -### 2>&1 | FileCheck %s -DOPTION=-fno-sycl-use-bitcode
 // RUN: %clangxx -fsycl -fsycl-allow-device-dependencies %s -### 2>&1 | FileCheck %s -DOPTION=-fsycl-allow-device-dependencies
 // RUN: %clangxx -fsycl -fno-sycl-allow-device-dependencies %s -### 2>&1 | FileCheck %s -DOPTION=-fno-sycl-allow-device-dependencies
 // CHECK: option '[[OPTION]]' is deprecated and will be removed in a future release
+
+// RUN: %clangxx -fsycl -fsycl-use-bitcode %s -### 2>&1 \
+// RUN: | FileCheck %s --check-prefix=CHECK_REPLACE -DOPTION=-fsycl-use-bitcode -DOPTION_REPLACE=-fsycl-device-obj=llvmir
```

Comment:
These tests imply that `fsycl-use-bitcode` and `fno-sycl-use-bitcode` options are deprecated and replaced by `fsycl-device-obj=llvmir` and `fsycl-device-obj=spirv` respectively.

But the PR description only mentions about `fsycl-fp32-prec-sqrt`.
I also don't see the user manual updated for `fsycl-use-bitcode` and `fno-sycl-use-bitcode` options

Reply:
The `-fsycl-use-bitcode` behavior was a side effect of updating the diagnostic message and behavior.  I'll update description and user manual accordingly.